04.ls/ls.rb

```diff
@@ -2,15 +2,44 @@
 # frozen_string_literal: true
 
 require 'optparse'
+require 'etc'
 
+FILE_TYPE = {
+  'file' => '-',
+  'directory' => 'd',
+  'characterSpecial' => 'c',
+  'blockSpecial' => 'b',
+  'fifo' => 'p',
+  'link' => 'l',
+  'socket' => 's'
+}.freeze
+PERMISSION = {
```

Comment:
```suggestion

PERMISSION = {
```

複数行の定数定義の前後には空行を入れた方が読みやすいかも

Reply:
複数行の定数定義の前後に空行を入れました！